www_src/pages/element/text-editor.jsx

```diff
@@ -16,9 +16,16 @@ var TextEditor = React.createClass({
       fontWeight: 'normal',
       fontStyle: 'normal',
       textDecoration: 'none',
-      textAlign: 'center'
+      textAlign: 'center',
+      innerHTML: 'Hello world'
     };
   },
+  editText: function () {
```

Comment:
Ah, I see -- ok, I added it in. I wanted to preserve the fade in animation though if that's ok